### DISCUSSION TITLE
230724 UX 저자 워크숍 대비 소개글 작성 (13:40 ~ 15:00)

### DISCUSSION THREAD

**Discussion by Creative-Lee** [woowacourse-teams/2023-shook#102] [회의]
[준의 소개글 예시](https://techcourse.woowahan.com/s/Vv5RJJLe/ls/lJ2J4mdh)

7월 25일 까지 최대 1500자 미만의 프로젝트 소개글을 작성한다.

**Comment by Cyma-s** [comment]
# 서론
우리 서비스를 쓰면 좋은 상황 -> 이러신 적 없으신가요?

# 본론
S-HOOK이 무슨 서비스인지 설명 (기능을 예시 들면서 설명하기 - 시나리오)

# 결론
S-HOOk을 사용해서 이룰 수 있는 목표
